lib/galaxy/web/base/interactive_environments.py

```diff
@@ -438,6 +438,10 @@ def launch(self, image=None, additional_ids=None, env_override=None, volumes=Non
             raise Exception("Attempting to launch disallowed image! %s not in list of allowed images [%s]"
                             % (image, ', '.join(self.allowed_images)))
 
+        # Do not allow a None volumes
```

Comment:
Yeah - Nate broke it in a container interface refactor. I had the exact commit earlier but I lost it. No big deal - I don't think it is broken in release_17.01.